addons/web/static/tests/views/basic_model_tests.js

```diff
@@ -21,6 +22,7 @@ odoo.define('web.basic_model_tests', function (require) {
                         category: { string: "Category M2M", type: 'many2many', relation: 'partner_type' },
                         date: { string: "Date Field", type: 'date' },
                         reference: { string: "Reference Field", type: 'reference', selection: [["product", "Product"], ["partner_type", "Partner Type"], ["partner", "Partner"]] },
+                        x_active: { string: "Custom Active", type: 'boolean', default: true},
```

Comment:
Many JS implementations use the insertion order for keys in objects.  So, in order to check that `_getActiveField` returns the right one, you should put them in reverse order: `'x_active'`, `'active'`.